docker-cloud/builds/image-scan.md

```diff
@@ -1,171 +0,0 @@
----
-description: 'Docker Security Scanning: automatic image scanning'
```

Comment:
Should we make this URL redirect to (e.g.) DTR scanning? https://docs.docker.com/datacenter/dtr/2.4/guides/admin/configure/set-up-vulnerability-scans/

Reply:
Good point. I think it's going to be confusing if I land on a product that's completely unrelated to Docker Cloud.

Maybe during the deprecation period we should leave the article but add a message up in the top linking to the deprecation announcement or something that tells me that security scans are available in our enterprise offer?